econplayground/main/migrations/0097_alter_assignment_instructor.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.16 on 2022-12-20 18:42
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('main', '0096_assignment_instructor'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='assignment',
+            name='instructor',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL),
+        ),
+    ]
```

Comment:
So, this pull request is adding new migrations based on all the feedback I gave to the models, with AlterField, RemoveField etc, calls. It would be better to re-create the original migration file based on the current state of the models - there's no need for the database to go through all these changes as we were finalizing things.

I can clean this up pretty easily, so I'll make that change. It just entails removing all the migration files and running `./manage.py makemigrations`, as long as your database is in the basic `master` state.